test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NetworkCallCountTest.cs

```diff
@@ -1197,7 +1197,13 @@ public void NetworkCallCount_RestoreSolutionMultipleSourcesTwoV3()
                     slnPath,
                     "-Verbosity",
                     "detailed",
-                    "-DisableParallelProcessing"
+                    "-DisableParallelProcessing",
+                    "MSBuildVersion",
```

Comment:
It's a good point. This is the only test that breaks in dev14 for this reason, but it may make sense to be consistent. Will investigate.